api/provisioning/v1alpha1/provisioningrequest_types.go

```diff
@@ -43,30 +47,18 @@ type ProvisioningRequestSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Template Version",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	TemplateVersion string `json:"templateVersion"`
 
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Cluster Template Input",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
-	ClusterTemplateInput ClusterTemplateInput `json:"clusterTemplateInput"`
+	// TemplateParameters provides the input data that conforms to the schema defined in the referenced ClusterTemplate's spec.templateParameterSchema.
```

Comment:
nit, would be good to add that the template must be in openapi-v3 format

Reply:
The description of templateParameterSchema field in the ClusterTemplate mentions that the schema should be openAPI v3 format, but in the provisioningRequest I can change to this
```suggestion
	// TemplateParameters provides the input data that conforms to the OpenAPI v3 schema defined in the referenced ClusterTemplate's spec.templateParameterSchema.
```